src/components/List/src/listitem.css

```diff
@@ -0,0 +1,86 @@
+.denhaag-list__item {
+  padding-block-start: var(--denhaag-list-item-padding-block);
+  padding-block-end: var(--denhaag-list-item-padding-block);
+  padding-inline-start: var(--denhaag-list-item-padding-inline);
+  padding-inline-end: var(--denhaag-list-item-padding-inline);
+}
+
+.denhaag-list__item:focus {
```

Comment:
For the other items we added a modifier rule for all CSS states (`hover`, `focus`, `active`): `.denhaag-list__item--focus`.
This is done for easier visual regression testing which at this point we're not actually doing, but I still would like to see it added just in case.

Reply:
I've added this for the easy cases, others were not as easy.
For instance what rule should be used here:

`.denhaag-list__item[secondarytext]:active`

I've added an issue to resolve this in the future, and I think we can merge this PR.